api/src/test/java/org/openmrs/test/jupiter/BaseModuleActivatorTest.java

```diff
@@ -0,0 +1,56 @@
+/**
+ * This Source Code Form is subject to the terms of the Mozilla Public License,
+ * v. 2.0. If a copy of the MPL was not distributed with this file, You can
+ * obtain one at http://mozilla.org/MPL/2.0/. OpenMRS is also distributed under
+ * the terms of the Healthcare Disclaimer located at http://openmrs.org/license.
+ *
+ * Copyright (C) OpenMRS Inc. OpenMRS is a registered trademark and the OpenMRS
+ * graphic logo is a trademark of OpenMRS Inc.
+ */
+package org.openmrs.test.jupiter;
+
+import org.junit.jupiter.api.BeforeEach;
+import org.openmrs.module.ModuleConstants;
+import org.openmrs.module.ModuleTestData;
+import org.openmrs.module.ModuleUtil;
+
+/**
+ * Has functionality commonly used by unit tests for module activator
+ */
+public abstract class BaseModuleActivatorTest extends BaseContextSensitiveTest {
```

Comment:
@dkayiwa since I saw your name on the commiters of this file and the ModuleActivatorTest. Can you describe to us what these are testing?

The reason why I am asking is that for the JUnit 4 to 5 migration we need to figure out for each of these test helpers if they are only used internally within openmrs-core. This one is used in `ModuleActivatorTest` in org.openmrs.module and WebModuleActivatorTest in `package org.openmrs.web.test;.

If it is only used internally we can choose to switch to JUnit 5 without also providing a JUnit 4 version. If it is for example used by module devs we need to provide a JUnit 4 version so they have a transition period. So they can choose to migrate at their own pace.

@achilep I will wait for @dkayiwa before we make a decision on what to do with this test helper.

Reply:
ModuleActivatorTest and WebModuleActivatorTest are only used internally. So go ahead and switch to JUnit 5